worker/mocks_test.go

```diff
@@ -490,43 +500,41 @@ func (os *objectStoreMock) Slab(ctx context.Context, key object.EncryptionKey) (
 	return
 }
 
-func (os *objectStoreMock) UpdateSlab(ctx context.Context, s object.Slab, contractSet string) error {
+func (os *objectStoreMock) UpdateSlab(ctx context.Context, key object.EncryptionKey, sectors []api.UploadedSector) error {
 	os.mu.Lock()
 	defer os.mu.Unlock()
 
+	updated := make(map[types.Hash256]types.FileContractID)
```

Comment:
This is a bit sneaky. It means that if you pass the same sector multiple times this will silently fail which we might not want. Better to be explicit and return an error if a sector is specified more than once.